components/dashboard/src/projects/projects.routes.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
TL;DR: nobody ever, in any possible situation or personal condition ever, will have to think twice about which feature this file refers to (projects) or what it contains (routes).

This is something I started doing when I realised that in some contexts we don't usually think about, a file could be misinterpreted or colleagues I was sharing it with weren't immediately sure of which file it was and to which mental context they needed to switch in order to have a conversation about it. Whereas to me it was obvious what it was because I had just picked it and shared it.
Copy it around, or share it, or move it, or open it where its full path doesn't fit and you can only read its name, or ask a sleep deprived colleague to take a look, etc, etc.
Nobody ever will be unsure about what a file with a name like this is.
It lightens the cognitive load for developers.
CC @loujaybee because this is an oft neglected example of DevX 😉

Reply:
:+1: